README.md

```diff
@@ -316,57 +196,6 @@ $ sudo systemctl start station
 $ sudo systemctl status station
 ```
 
```

Comment:
Can we add a section describing how to inspect logs when running Station Core as a system service?